src/main/java/application/AbstractBaseController.java

```diff
@@ -289,6 +298,15 @@ public ExceptionResponseBean handleError(FormplayerHttpRequest req, Exception ex
         return new ExceptionResponseBean(exception.getMessage(), req.getRequestURL().toString());
     }
 
+    private void incrementDatadogCounter(String metric, FormplayerHttpRequest req) {
+        datadogStatsDClient.increment(
+                metric,
+                "domain:" + req.getDomain(),
+                "user:" + req.getCouchUser().getUsername(),
```

Comment:
presumably serverup will never go through error handling, but maybe we should guard against that?